src/features/embed-gist-inline.js

```diff
@@ -0,0 +1,29 @@
+import select from 'select-dom';
+
+const gistRegex = /gist\.github\.com\/\w*?\/\w*/;
+const isGist = link => gistRegex.test(link.href);
```

Comment:
Use https://github.com/sindresorhus/refined-github/blob/a8fd502d23084044c282bcdf56bac92666c2238f/src/libs/page-detect.js#L42

Reply:
This checks the current location. Do you mean to recreate the host or path check?

Reply:
Never mind. I misread the code